Source/WebCore/page/SecurityOrigin.cpp

```diff
@@ -176,7 +176,14 @@ bool shouldTreatAsPotentiallyTrustworthy(const URL& url)
 }
 
 SecurityOrigin::SecurityOrigin(const URL& url)
-    : m_data(SecurityOriginData::fromURL(url))
+    : SecurityOrigin(SecurityOriginData::fromURL(url))
+{
+    if (m_canLoadLocalResources)
+        m_filePath = url.fileSystemPath(); // In case enforceFilePathSeparation() is called.
+}
+
+SecurityOrigin::SecurityOrigin(const SecurityOriginData& data)
```

Comment:
Why not `SecurityOriginData&&` and `WTFMove(data)` below? Is there a call site that does't pass a r-value reference?